examples/03_connectivity/plot_simulated_connectome.py

```diff
@@ -25,11 +19,20 @@ def plot_matrix(m, ylabel=""):
     n_subjects=n_subjects, n_features=10, min_n_samples=30, max_n_samples=50,
     density=0.1)
 
+from nilearn.plotting.matrix_plotting import plot_matrix
+
+
+def plot_matrix_symmetric_scale(mat, **kwargs):
+    abs_max = abs(mat.max())
```

Comment:
It seems surprising to me that the "abs" is needed: I would expect the max of the matrix to be positive.

Given this, I would suggest to get rid of this intermediate function "plot_matrix_symmetric_scale", and directly inline the code and call "plot_matrix" in the example, to make things simpler.